src/metrics/cyclomatic.rs

```diff
@@ -80,10 +109,12 @@ impl Cyclomatic for PythonCode {
         match node.object().kind_id().into() {
             If | Elif | For | While | Except | With | Assert | And | Or => {
                 stats.cyclomatic += 1.;
+                stats.self_cyclomatic += 1.;
```

Comment:
You compute the same thing two times: `stats.cyclomatic += 1.;` and `stats.self_cyclomatic += 1.;`: maybe having a `self.self_cyclomatic = self.cyclomatic` at the right place would simplify things.

Reply:
I will check if that is possible, but last time I checked I didn't find a way to avoid this change.